src/Web/Rollbar/Class.hs

```diff
@@ -0,0 +1,25 @@
+module Web.Rollbar.Class
+    ( MonadRollbarWriter(..)
+    ) where
+
+import qualified Web.Rollbar as R
+import Control.Monad.Except (MonadError)
+import Control.Monad.Reader (MonadReader)
+import Control.Monad.Trans (MonadIO)
+import Network.HTTP.Nano
+    ( AsHttpError
+    , HasHttpCfg
+    )
+
+class MonadRollbarWriter m where
+    rollbar :: (R.ToRollbarEvent evt) => evt -> m ()
+
+instance 
```

Comment:
I think this is going to cause some Overlapping Instances problems, but I'm not sure how to provide this instance so that the vines themselves don't all have to.

Reply:
Well, maybe it shouldn't be in this library at all? Perhaps `cv-base` would be a better place?

Reply:
Good idea! I've done so there, so I think I'm going to close this PR.